bootstrap-salt.sh

```diff
@@ -6026,6 +6030,23 @@ config_salt() {
         fi
     fi
 
+    if [ "$_INSTALL_CLOUD" -eq $BS_TRUE ]; then
```

Comment:
Hrm...I am not sure if the bootstrap script should be responsible for something like this. The rest of your changes look great, but when I got to this block, it seems like it is outside of the scope of "install the salt-cloud package along with libcloud when I pass `-L`"

I'd like @s0undt3ch to weigh in on this matter before we merge this.